### PR TITLE
Just a few updates for version, README and Berks

### PR DESCRIPTION
**Pull request by Brantone**
Update attributes to latest NRM version, fix README reference to cookbook name, update gitignore and remove Berksfile.lock

### CODE REVIEW

File: Berksfile.lock [hunk 1]
```diff
@@ -1,33 +0,0 @@
-DEPENDENCIES
```
Comment (whyjustin) [inline Berksfile.lock]: Why was this removed?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,54 @@
-.idea
-.kitchen
 .DS_Store
 build/
+
+*.rbc
```
Comment (whyjustin) [inline .gitignore]: Are these all required? I'd rather keep .gitignore lean to expose when new files are being brought into a workspace which can highlight their ramifications to the project.

File: README.md [hunk 3]
```diff
@@ -2,20 +2,20 @@
 
 #### Usage
 
-Simply include the `nexus-repository-manager` recipe wherever you would like Nexus Repository Manager installed. Simply add
-`recipe['nexus-repository-manager']` to your runlist or `include_recipe 'nexus-repository-manager'` to your cookbook. This will
+Simply include the `nexus_repository_manager` recipe wherever you would like Nexus Repository Manager installed. Simply add
```
Comment (whyjustin) [inline README.md]: Thanks for catching this 👍

File: .gitignore (hunk 2 above)
Comment (Brantone) [reply .gitignore]: Most of 'em are added by default when running `chef generate cookbook` , some others are future-proofing. I could easily pare it down to match Chef defaults.

File: Berksfile.lock (hunk 1 above)
Comment (Brantone) [reply Berksfile.lock]: Partly because it's default with Chef (per other comment: when running `chef generate cookbook`), but also because of (a.) wrapper cookbooks, and/or (b.) policyfiles ... bit of a shift in the community. I can back this out, if preferred.

Comment (whyjustin) [reply Berksfile.lock]: I don't know enough about chef to comment but we need a way to have reproducible builds on every machine (CI/dev) so this can't be removed until that problem has been solved. It would be outside the scope of this change either way.

File: .gitignore (hunk 2 above)
Comment (whyjustin) [reply .gitignore]: This reminds me of the discussion here [0]. I would prefer only adding entries in .gitignore for files that are created during the development process that should not be committed to source control. This may mean omitting some that chef creates by default.

[0] https://github.com/sonatype/chef-nexus-repository-manager/pull/1#discussion_r137826127

File: Berksfile.lock (hunk 1 above)
Comment (Brantone) [reply Berksfile.lock]: Understandable on the reproducibility angle ... any chance of getting this cookbook hooked up to something like travis-ci for builds (ex: have a look at https://github.com/chef-cookbooks/nginx)
All that being said, because metadata.rb has pessimistic versioning, there's no guarantee that subsequent `berks update` runs will be identical anyway.
I'll update the PR for now.

File: .gitignore (hunk 2 above)
Comment (Brantone) [reply .gitignore]: Even if those defaults are prescribed, based on extensive development, by the upstream creators? ex: Chef? Some of the ignore entries are also future-proofing.

Comment (whyjustin) [reply .gitignore]: Or they have an opinionated process which is defensive against letting non source files into GitHub whereas we have one which is defensive against misunderstood intent. /me shrugs.

File: Berksfile.lock (hunk 1 above)
Comment (whyjustin) [reply Berksfile.lock]: We have gone down the avenue of using travis for Sonatype projects and it usually runs into headaches maintaining an internal Jenkins build and external travis build. I need to think more about the benefit of feedback to external contributors against the maintenance requirements.

**Comment by whyjustin** [comment]
> To avoid holding up the other changes while on-going discussion continues, I've split out just the tweaks to attributes & README.md into separate PR: #33

Thanks, going to close this one as only the contentious changes remain.
